destroy_cluster.sh

```diff
@@ -36,9 +43,11 @@ done
 
 declare -r installer="${OPENSHIFT_INSTALLER:-openshift-install}"
 
-# Remove entries from /etc/hosts and ssh config
-sudo sed -i "/# Generated by shiftstack for $CLUSTER_NAME - Do not edit/,/# End of $CLUSTER_NAME nodes/d" /etc/hosts
-sed -i "/# Generated by shiftstack for $CLUSTER_NAME - Do not edit/,/# End of $CLUSTER_NAME nodes/d" $HOME/.ssh/config
+if [[ $SET_CONFIG -gt 0 ]]; then
```

Comment:
Have you considered dynamically skip removing the entries from `/etc/hosts` if we find that we don't have write permission, for example with `test -w` ? I prefer we don't make the script more complex with undocumented flags.

Reply:
Good tip: that works for `~/.ssh/config`. However I am not sure how to make it work for /etc/hosts: the current user will likely not have write permissions to /etc/hosts. Shall I check if he's in the wheel group? Sounds a bit shady to change logic based on group affiliation.